shell/probe/raindrop.ps1

```diff
@@ -23,33 +23,37 @@ function FromEnv { param ([string]$envVar, [string]$default)
 
 $Dir = FromEnv "PRELUDE_DIR" ".vst"
 $Sleep = FromEnv "PRELUDE_SLEEP" 14440
-$CA = "prelude-account-prod-us-west-1.s3.amazonaws.com"
+$CA = "prelude-account-us1-us-west-1.s3.amazonaws.com"
 
 $Api = "https://api.preludesecurity.com"
 $Dos = "windows-$Env:PROCESSOR_ARCHITECTURE"
 $Dat = ""
 
 while ($true) {
     try {
-        $Vst = New-Item -Path "$Dir\$(New-Guid).exe" -Force
         $Headers = @{
             'token' = FromEnv "PRELUDE_TOKEN"
             'dos' = $Dos
             'dat' = $Dat
-            'version' = "1.0"
+            'version' = "1.1"
+        }
+
+        $Redirect = Invoke-WebRequest -URI $Api -UseBasicParsing -Headers $Headers -MaximumRedirection 0
```

Comment:
I'm a bit worried messing with this too quickly; I'd recommend doing this server-side, in an "https://preludesecurity.com/upgrade" path